explore/src/main/scala/explore/config/AdvancedConfigurationPanel.scala

```diff
@@ -108,6 +113,9 @@ sealed abstract class AdvancedConfigurationPanelBuilder[
 ] {
   type AA = Aligner[T, Input]
 
+  @inline protected def isCustomized(aligner:         AA)(implicit ctx: AppContextIO): Boolean
```

Comment:
What does it mean `inline` when you'r defining an abstract method?

Reply:
Well, to be honest, I was just copying the pattern from the original AdvancedConfigurationPanel. I think (hope) it means that in the concrete classes the method will be inlined. But, I can't say if it actually works or not. The compiler doesn't complain. 🤷